BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee

```diff
@@ -1,8 +1,9 @@
 ContentTypeFormView = FullPageFormView.extend(
 
   onViewReady: ->
-    displayMenu() if @options.submitted
-
+    if @options.submitted
+      displayRoute = appRouter.generateUrl('listEntities', entityType: @options.entityType)
+      displayMenu(undefined, displayRoute)
```

Comment:
Ok, I see better now. I find weird to call a function with a route param sometimes as first argument, and sometimes as second, but in this case with no first parameter. Maybe you could change the function signature into something like ``displayMenu = (route, disableNavigate)``, with ``disableNavigate`` being a boolean.
That way the route is always the first parameter, and if required you can force to disable the ``Backbone.history.navigate``